Makefile

```diff
@@ -2,22 +2,58 @@
 #
 # It's auto-generated by sonata-project/dev-kit package.
 
-.PHONY: test docs
-
 all:
 	@echo "Please choose a task."
+.PHONY: all
+
+lint: lint-composer lint-yaml lint-composer lint-xml lint-php
+.PHONY: lint
 
-lint:
+lint-composer:
 	composer validate
+.PHONY: lint-composer
+
+lint-yaml:
 	find . -name '*.yml' -not -path './vendor/*' -not -path './Resources/public/vendor/*' | xargs yaml-lint
-	find . \( -name '*.xml' -or -name '*.xliff' \) \
-		-not -path './vendor/*' -not -path './Resources/public/vendor/*' \
-		| xargs -I'{}' xmllint --encode UTF-8 --output '{}' --format '{}'
+
+.PHONY: lint-yaml
+
+lint-xml:
+	find src \( -name '*.xml' -or -name '*.xliff' \) \
+		-not -path './src/Resources/public/vendor/*' \
+		| while read xmlFile; \
+	do \
+		XMLLINT_INDENT='    ' xmllint --encode UTF-8 --format "$$xmlFile"|diff - "$$xmlFile"; \
```

Comment:
Too bad Travis does not have the `--color` diff option... maybe on the 7.1 image?